src/service/programService.js

```diff
@@ -1016,6 +1018,50 @@ function programList(req, response) {
     res_limit = (data.request.limit < queryRes_Max) ? data.request.limit : (queryRes_Max);
   }
 
+  const filtersOnConfig = ['medium', 'subject', 'gradeLevel'];
+  const filters = {};
+  filters[Op.and] = _.map(data.request.filters, (value, key) => {
+    const res = {};
+    if (filtersOnConfig.includes(key)) {
+      res[Op.or] = _.map(data.request.filters[key], (val) => {
+        delete data.request.filters[key];
+        return {
+          ['config.' + key] : {
+            [Op.like]: '%' + val +'%'
+          }
+        };
+      });
+      return res;
+    }
+    else if (key === 'content_types' && value) {
+      res[Op.or] = _.map(data.request.filters[key], (val) => {
+        return Sequelize.literal(`'${val}' = ANY (content_types)`);
+      });
+      delete data.request.filters[key];
+      return {
+        $and : res
+      }
+    }
+    else if ((key === 'nomination' || key === 'contribution') && value) {
```

Comment:
have nomination_enddate and content_submission_enddate as the filters

Reply:
Done